Bugfixes

niwinz opened the pull request.

Alotor committed:
:bug: Fixed problem with import SVG image size

elhombretecla committed:
:tada: Add search section title

elhombretecla committed:
:sparkles: Add link to feedback dashboard

elhombretecla committed:
:tada: Add feedback link to workspace

elhombretecla committed:
:sparkles: Fix sidebar css

niwinz committed:
:tada: Add navigation to feature slides on the onboarding modal.

niwinz committed:
:bug: Remove duplicated translation string.

niwinz committed:
:bug: Show onboarding just after logging.

niwinz committed:
:sparkles: Increase heap memory on devenv repl script.

niwinz committed:
:bug: Avoid unexpected error when a cookie is expired.

niwinz committed:
:bug: Fix incorrect erorr reporting.

niwinz committed:
:bug: Refactor copy/paste for proper handle image shape copying.

Alotor commented on frontend/src/app/main/data/workspace.cljs:
```diff
@@ -1197,10 +1224,16 @@
+                        :objects {}
+                        :images #{}}]
+          (->> (rx/from (seq (vals pdata)))
+               (rx/fmap (partial prepare-object objects selected))
```
I think using the rxjs name  `merge-map` it's better for legibility. I did have to go to the source code to see what `fmap` meant.

niwinz committed:
:sparkles: Only print version on browser execution context.

niwinz committed:
:sparkles: Don't duplicate images when copy and paste in the same file.

niwinz committed:
:recycle: Refactor file-media-gc task (mainly add more traces).

niwinz committed:
:recycle: Minor task naming and directory structure refactor.

niwinz committed:
:tada: Add basic code for svg parsing to clj data structure.

Usage example:

curl -X POST http://localhost:6060/api/svg -H "content-type: image/svg+xml" -d "@example2.svg" |jq

niwinz committed:
:fire: Remove unused code.

niwinz committed:
:tada: Add first helpers for manipulate the file data from server repl.

niwinz committed:
:arrow_up: Update devenv.

niwinz committed:
:fire: Remove commented code.

niwinz committed:
:paperclip: Minor changes on manage.sh script.

niwinz committed:
:sparkles: Many improvements on error reporting.

niwinz committed:
:arrow_up: Update potok to 3.0.0.
